src/Test/QuickCheck/Checkers.hs

```diff
@@ -97,18 +97,21 @@ unbatch :: TestBatch -> [Test]
 unbatch (batchName,props) = map (first ((batchName ++ ": ")++)) props
 
 -- TODO: consider a tree structure so that flattening is unnecessary.
+type QuickCheckRunner = Args -> Property -> IO ()
```

Comment:
Right now it looks like the TODO refers to the new type synonym.


```suggestion

type QuickCheckRunner = Args -> Property -> IO ()
```

Reply:
Ah, yeah, that makes a lot of sense!